backend/osmosewebsite/models/trap.py

```diff
@@ -0,0 +1,28 @@
+"""Trap-related models"""
+
+from django.db import models
+from django.contrib.postgres import fields
+from tinymce.models import HTMLField
+from .team_member import TeamMember
+
+
+class Trap(models.Model):
+    """
+    Table containing Trap will be used on the website
+    """
+
+    firstname = models.CharField(max_length=100, default="DefaultFirstName")
```

Comment:
The first name and last name are specifically for the presenter not for the  TRAP 
Regarding the default value for the first name, it was implemented in response to a database constraint that generated an error when a value was not provided

Reply:
I know, it was a dumb question. As I told you, the name of this field should reflect your explanation. The name of the fields, classes, functions, etc should be cristal clear to facilitate the code comprehension.
All the field that describe the presenter must be renamed.

Keeping incorrect code because there is an error is never an appropriate answer. Either the field doesn't need to be mandatory so there is no need for default value (as for lastname in fact), either we find an other mean to handle migration so this mandatory field without default value is not a problem.

Reply:
I changed the name of firstname/lastname to lastname_presenter/firstname_presenter.

Reply:
Be careful the change hasn't been made in the fixtures, tests and in the react code (this appy for linkedin_url and the other field regarding presenter)